tools/vendor_hid_test.py

```diff
@@ -0,0 +1,259 @@
+"""These tests verify the functionality of the VendorHID interface."""
+import hid
+import time
+import unittest
+
+_OPENSK_VID = 0x1915
+_OPENSK_PID = 0x521F
+_FIDO_USAGE_PAGE = 0xF1D0
+_VENDOR_USAGE_PAGE = 0xFF00
+_PACKETS = 4
+_PACKET_SIZE = 64
+_SEND_DATA_SIZE = _PACKET_SIZE + 1
+_DEFAULT_CID = bytes([0xFF, 0xFF, 0xFF, 0xFF])
+
+
+def sleep():
+  #time.sleep(.01)
+  pass
```

Comment:
If we're robust enough to get away without sleeps, feel free to remove it.

Reply:
Not ready to get rid of this just yet.

Subsequent PRs will finish this up.